### PR TITLE
add sandbox_url option

### PR DESCRIPTION
**Pull request by derpierre65**
add an option to change the sandbox_url in `%APPDATA%\obs-studio\plugin_config\own3d`, can set with `"sandbox_url": "https://my-sandbox.url",` if not set the default sandbox_url are used.

### CODE REVIEW

**Comment by ghostzero** [comment]
Please provide a documentation example (how to use it).

**Review by ghostzero — CHANGES_REQUESTED** [review]
Please provide a documentation example. So we can maintain the [developer documentation](https://github.com/own3d/own3dpro-obs-plugin/wiki/Developer-Mode). This PR also has no description.

**Review by ghostzero — APPROVED** [review]
Looks now good to me

File: source/plugin.cpp [hunk 1]
```diff
@@ -190,7 +200,7 @@ std::string own3d::get_api_endpoint(std::string_view const args)
 std::string own3d::get_web_endpoint(std::string_view const args /*= ""*/)
 {
 	constexpr std::string_view URL_ENDPOINT         = "https://own3d.pro/";
-	constexpr std::string_view URL_ENDPOINT_SANDBOX = "https://sandbox.own3d.pro/";
+	std::string_view           URL_ENDPOINT_SANDBOX = get_sandbox_url();
```
Comment (Xaymar) [inline source/plugin.cpp:203]: std::string_view requires constant memory. std::string is deallocated before it is used, and will cause unexpected behavior.

**Review by Xaymar — CHANGES_REQUESTED** [review]

**Pull request merged**